configure.ac

```diff
@@ -1510,10 +1510,10 @@ AC_CONFIG_FILES([Makefile
                  tests/API/XCCDF/variable_instance/Makefile
 
                  tests/schemas/Makefile
-		tests/bz2/Makefile
-		tests/codestyle/Makefile
-		tests/oval_details/Makefile
-		tests/nist/Makefile
+                 tests/bz2/Makefile
+                 tests/codestyle/Makefile
+                 tests/oval_details/Makefile
+                 tests/nist/Makefile
 
                  src/SCE/Makefile
                  tests/sce/Makefile])
```

Comment:
Ok sorry.  I just thought I was fixing it because it was a mix of tabs and spaces.